api/HTMLElement.json

```
@@ -1581,6 +1581,45 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "ignores_find_in_page": {
+          "__compat": {
+            "description": "Element is ignored for the purposes of find-in-page.",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLElement/inert",
+            "spec_url": "https://html.spec.whatwg.org/multipage/interaction.html#dom-inert",
+            "tags": [
+              "web-features:inert"
+            ],
+            "support": {
+              "chrome": {
+                "version_added": "124"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "124"
```

See: https://bugzilla.mozilla.org/show_bug.cgi?id=1868316

```suggestion
                "version_added": "120"
```